source/nodejs/adaptivecards/src/rendercard.ts

```diff
@@ -13,15 +13,11 @@ export function renderCard(card: IAdaptiveCard | string, options?: RenderOptions
     if (typeof card === "string") {
         card = <IAdaptiveCard>JSON.parse(card);
     }
-    
-    options = options || {};
 
-    let adaptiveCard = new AdaptiveCard();
-    adaptiveCard.parse(card);
-    adaptiveCard.onAction = options.onAction;
+    options = options || {};
 
+    // Parse the host config
     let hostConfig: HostConfig;
-
     if (isHostConfig(options.hostConfig)) {
```

Comment:
This seems weak. I don't think the isHostConfig method is necessary. Should be something like this:
```
if (typeof hostConfig === "string") {
    hostConfig = new HostConfig(JSON.parse(hostConfig));
}
else if (typeof hostConfig === "object" && !(hostConfig instanceof HostConfig)) {
    hostConfig = new HostConfig(hostConfig);
}
```